int32 src_uav_id
string component
int32 cmd_priority
string playload
float64 start_time
int64 seq_num
float64 time_stamp
int32 tgt_uav_count